Blimp/Parameters.cpp

```diff
@@ -102,7 +102,7 @@ const AP_Param::Info Blimp::var_info[] = {
     // @Param: FS_GCS_ENABLE
     // @DisplayName: Ground Station Failsafe Enable
     // @Description: Controls whether failsafe will be invoked (and what action to take) when connection with Ground station is lost for at least 5 seconds. See FS_OPTIONS param for additional actions, or for cases allowing Mission continuation, when GCS failsafe is enabled.
-    // @Values: 0:Disabled/NoAction,1:RTL,2:RTL or Continue with Mission in Auto Mode (Removed in 4.0+-see FS_OPTIONS),3:SmartRTL or RTL,4:SmartRTL or Land,5:Land (4.0+ Only)
+    // @Values: 0:Disabled/NoAction,5:Land (4.0+ Only)
```

Comment:
Perhaps we could remove the "4.0+ only" since Blimp doesn't have a pre-4.1 in any case?

Reply:
no problem